frontend/src/pages/GraphPF/GraphPFElems.tsx

```diff
@@ -170,7 +170,7 @@ export const setNodeAttachments = (node: Node<NodeModel>, settings: GraphPFSetti
   const data = node.getData() as NodeData;
   const attachments = [] as React.ReactNode[];
 
-  if (settings.showMissingSidecars && data.hasMissingSC) {
+  if (settings.showMissingSidecars && data.isOutOfMesh) {
```

Comment:
We should probably also change the setting name.

Reply:
That's right. I updated all the occurrences.